src/components/Restaurant/Restaurant.test.jsx

```diff
@@ -14,9 +14,27 @@ describe("Restaurant", () => {
 
   it("increments the number of ordered pizzas when 'Add' button is clicked", () => {
     render(<Restaurant />);
-    const addButton = screen.getByText("Add");
+    const pizzasElement = screen.getByText(/Pizzas/);
+    const addButton = pizzasElement.querySelector(".restaurant__button");
     fireEvent.click(addButton);
-    const pizzasElement = screen.getByText("Pizzas: 1");
-    expect(pizzasElement).toBeInTheDocument();
+    const pizzasElementAfterClick = screen.getByText("Pizzas: 1");
+    expect(pizzasElementAfterClick).toBeInTheDocument();
+  });
+  it("increments the number of ordered salads when 'Add' button is clicked", () => {
+    render(<Restaurant />);
+    const saladsElement = screen.getByText(/Salads/);
+    const addButton = saladsElement.querySelector(".restaurant__button");
+    fireEvent.click(addButton);
+    const saladsElementAfterClick = screen.getByText("Salads: 1");
+    expect(saladsElementAfterClick).toBeInTheDocument();
+  });
+  it("increments the number of ordered chocolate cake when 'Add' button is clicked", () => {
+    render(<Restaurant />);
+    const chocolateCakeElement = screen.getByText(/Chocolate cake/);
+    const addButton = chocolateCakeElement.querySelector(".restaurant__button");
+    fireEvent.click(addButton);
+    const chocolateCakeElementAfterClick =
+      screen.getByText("Chocolate cake: 1");
+    expect(chocolateCakeElementAfterClick).toBeInTheDocument();
   });
```

Comment:
Clever implementation fo the tests and well done on correcting the failed error on tests. 
Now I understand how you did that.